### PR TITLE
OSDOCS-4302: Adds notes for 4.11.8 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for 4.11.8 release

Version(s):
4.11 only

Issue:
[OSDOCS-4302](https://issues.redhat.com/browse/OSDOCS-4302)

Link to docs preview:
2100860 in the known issues section: https://51395--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues

4.11.8: https://51395--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-8

QE review:
- [ ] QE has approved this change. 
* QE not needed for this change

Additional information:
Links will not work yet

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -2344,9 +2344,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
-* In the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the *Developer* perspective in the {product-title} web console.
-No workaround exists for this issue.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=2100860[*BZ#2100860*])
+* In the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the *Developer* perspective in the {product-title} web console. This issue has been fixed in 4.11.8. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100860[*BZ#2100860*])
```
Comment (Wiharris) [inline release_notes/ocp-4-11-release-notes.adoc:2347]: Wanted to ask if there's a reason why "Developer" in developer perspective is bold and capitalized? I have not seen it as such elsewhere in docs, but if it is apologize! :)

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -2867,4 +2865,28 @@ $ oc adm release info 4.11.7 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.11.8 --pullspecs
+----
+
+[id="ocp-4-11-8-bug-fixes"]
+==== Bug fixes
+
+* Previously, in the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the Developer perspective in the OpenShift Container Platform web console. With this update, user-defined alerts can be silenced from the Developer perspective. (link:https://issues.redhat.com/browse/OCPBUGS-1790[*OCPBUGS-1790*])
```
Comment (Wiharris) [inline release_notes/ocp-4-11-release-notes.adoc:2887]: Same question as above in terms of "Developer" being capitalized.

Comment (Wiharris) [inline release_notes/ocp-4-11-release-notes.adoc:2887]:
```suggestion
* Previously, in the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the Developer perspective in the {product-title} web console. With this update, user-defined alerts can be silenced from the Developer perspective. (link:https://issues.redhat.com/browse/OCPBUGS-1790[*OCPBUGS-1790*])
```

**Review by Wiharris — COMMENTED** [review]
/LGTM! Left a minor question and one minor suggestion below! :)

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:2887]: See comment above

Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:2887]: Whoops! Thanks for catching. I missed with copy/paste.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (Wiharris) [reply release_notes/ocp-4-11-release-notes.adoc:2347]: Got ya I see now! Thanks!

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (Wiharris) [reply release_notes/ocp-4-11-release-notes.adoc:2887]: No problem!

**Pull request merged**